tcf_website/views/feedback.py

```diff
@@ -0,0 +1,105 @@
+    else:
+        return JsonResponse({'ok': False, 'content': message})
+
+
+def send_email(subject, message, from_email, recipient_list):
+    """Send email"""
+    try:
+        send_mail(subject, message, from_email, recipient_list)
+    except BadHeaderError:
+        return JsonResponse({'ok': False, 'content': message})
+    else:
+        return JsonResponse({'ok': True, 'content': message})
+
+
+@require_http_methods('POST')
+def discord_feedback(query):
+    """Prepare feedback for discord message"""
+    message_type = query.POST.get("type", "")
+    content = json.loads(query.POST.get("content", ""))
+
+    if message_type == "bug":
+        url = os.environ['DISCORD_URL_BUG']
+        message = textwrap.dedent(f"""\
+            Bug Found!
+            **URL:** {content["url"]}
+            **Categories:** {content["categories"]}
+            **Email:** {content["email"]}
+            **Description:** {content["description"]}""")
+        return send_discord(url, message)
+
+    if message_type == "feedback":
+        url = os.environ['DISCORD_URL_FEEDBACK']
+        message = textwrap.dedent(f"""\
+            Feedback submitted
+            **Name:** {content["fname"]} {content["lname"]}
+            **Email:** {content["email"]}
+            **Title:** {content["title"]}
+            **Message:** {content["message"]}""")
+        return send_discord(url, message)
+    return JsonResponse({'ok': False, 'content': query})
+
+
+@require_http_methods('POST')
+def email_feedback(query):
+    """Prepare feedback for email"""
+    message_type = query.POST.get("type", "")
+    content = json.loads(query.POST.get("content", ""))
+
+    from_email = os.environ['EMAIL_HOST_USER']
+    to_email = content["email"]
+    recipient_list = [
+        to_email,
+        from_email
+    ]
+    subject = "[theCourseForum] Thank you for your feedback!"
+
+    if message_type == "bug":
+        message = textwrap.dedent(f"""\
+            Thanks for reaching out!
+
+            We received the following bug report from you
+            Description: {content["description"]}
+            Categories: {content["categories"]}
+
+            We apologize for any inconveniences that this may have caused.
+            Our team will be investigating the issue and will follow up with you shortly.
+
+            Best,
+            theCourseForum Team""")
+        return send_email(subject, message, from_email, recipient_list)
+
+    if message_type == "feedback":
+        message = textwrap.dedent(f"""\
+            Hi {content["fname"]},
+
+            Thanks for reaching out! We received the following feedback from you
+            Title: {content["title"]}
+            Message: {content["message"]}
+
+            We greatly appreciate you taking the time to help us improve tCF!
+            A team member will be following up with you shortly if neccesary.
+
+            Best,
+            theCourseForum Team""")
```

Comment:
It's probably fine for now, but seems a little clunky to have the email template hard coded if we need to make edits in the future.